taxi/views.py

```diff
@@ -87,3 +88,47 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(View):
+    def get(self, request):
+        form = DriverCreationForm()
+        return render(request,
+                      "taxi/driver_list.html", {"form": form})
```

Comment:
The template name 'taxi/driver_list.html' seems incorrect for rendering the driver creation form. It would be more appropriate to use a template like 'taxi/driver_form.html' to match the context of creating a new driver.